app/src/common/shared/org/mozilla/vrbrowser/browser/tracking/TrackingProtectionPolicy.java

```diff
@@ -0,0 +1,133 @@
+package org.mozilla.vrbrowser.browser.tracking;
+
+import androidx.annotation.IntDef;
+
+import org.mozilla.geckoview.ContentBlocking;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class TrackingProtectionPolicy {
+
+    @IntDef(value = { ETP_NONE, ETP_DEFAULT, ETP_STRICT})
+    public @interface ETP_Level {}
+    public static final int ETP_NONE = ContentBlocking.EtpLevel.NONE;
+    public static final int ETP_DEFAULT = ContentBlocking.EtpLevel.DEFAULT;
+    public static final int ETP_STRICT = ContentBlocking.EtpLevel.STRICT;
+
+    private List<TrackingCategory> trackingCategory;
+    private CookiePolicy cookiePolicy;
+    private Boolean strictSocialTrackingProtection = null;
+
+    private TrackingProtectionPolicy() {
+        trackingCategory = new ArrayList<>();
+    }
+
+    /**
+     * Strict policy.
+     * Combining the [TrackingCategory.STRICT] plus a cookiePolicy of [ACCEPT_NON_TRACKERS]
+     * This is the strictest setting and may cause issues on some web sites.
+     */
+    public static TrackingProtectionPolicy strict() {
+        TrackingProtectionPolicy policy = new TrackingProtectionPolicy();
+        policy.trackingCategory = Collections.singletonList(TrackingCategory.STRICT);
+        policy.cookiePolicy = CookiePolicy.ACCEPT_NON_TRACKERS;
+        policy.strictSocialTrackingProtection = true;
+        return policy;
+    }
+
+    /**
+     * Recommended policy.
+     * Combining the [TrackingCategory.RECOMMENDED] plus a [CookiePolicy] of [ACCEPT_NON_TRACKERS].
+     * This is the recommended setting.
+     */
+    public static TrackingProtectionPolicy recommended() {
+        TrackingProtectionPolicy policy = new TrackingProtectionPolicy();
+        policy.trackingCategory = Collections.singletonList(TrackingCategory.RECOMMENDED);
+        policy.cookiePolicy = CookiePolicy.ACCEPT_NON_TRACKERS;
+        policy.strictSocialTrackingProtection = false;
+        return policy;
+    }
+
+    public static TrackingProtectionPolicy none() {
+        TrackingProtectionPolicy policy = new TrackingProtectionPolicy();
+        policy.trackingCategory = Collections.singletonList(TrackingCategory.NONE);
+        policy.cookiePolicy = CookiePolicy.ACCEPT_ALL;
+        return policy;
+    }
+
+    public enum TrackingCategory {
+        NONE(0),
+        AD(1 << 1),
+        ANALYTICS(1 << 2),
+        SOCIAL(1 << 3),
+        CONTENT(1 << 4),
+        TEST(1 << 5),
+        CRYPTOMINING(1 << 6),
+        FINGERPRINTING(1 << 7),
+        MOZILLA_SOCIAL(1 << 8),
+        SCRIPTS_AND_SUB_RESOURCES(1 << 9999),
+        RECOMMENDED(AD.value + ANALYTICS.value + SOCIAL.value +
+                TEST.value + MOZILLA_SOCIAL.value + CRYPTOMINING.value),
+        STRICT(RECOMMENDED.value + SCRIPTS_AND_SUB_RESOURCES.value +
+                FINGERPRINTING.value);
```

Comment:
And this, should we just use the values in GeckoView so if they change we don't need to keep them synchronized?